servers/rendering/renderer_rd/shader_rd.cpp

```diff
@@ -149,7 +149,7 @@ void ShaderRD::setup(const char *p_vertex_code, const char *p_fragment_code, con
 
 	StringBuilder tohash;
 	tohash.append("[GodotVersionNumber]");
-	tohash.append(VERSION_NUMBER);
+	tohash.append(GODOT_VERSION_NUMBER);
 	tohash.append("[GodotVersionHash]");
 	tohash.append(VERSION_HASH);
```

Comment:
Missed this one:
```suggestion
	tohash.append(GODOT_VERSION_HASH);
```
I suggest testing a compile without the deprecated aliases to see if there aren't others that were missed.